src/_includes/assets/scripts/copy-links-for-usage-tab.ts

```diff
@@ -0,0 +1,22 @@
+window.addEventListener('DOMContentLoaded', function() {
+    const headers = document.querySelectorAll('#usage h1, #usage h2, #usage h3, #usage h4');
+
+    function getUrlNoHash() {
+        return window.location.href.includes("#")
+            ? window.location.href.split("#")[0]
+            : window.location.href;
+    }
```

Comment:
You might not need the `includes` check:

```suggestion
    function getUrlNoHash() {
        return window.location.href.split("#")[0];
    }
```